code/game/gamemodes/game_mode.dm

```diff
@@ -541,3 +546,25 @@ proc/get_nt_opposed()
 			to_chat(usr, "[ticker.mode.extended_round_description]")
 	else
 		to_chat(usr, "<i>Shhhh</i>. It's a secret.")
+
+//Handlers for people dying / being revived.
+/datum/game_mode/proc/on_crew_death(mob/living/carbon/human/H)
+	if(!H?.client)
```

Comment:
i'm not sure this would even compile with that questionmark, presumably a typo
But regardless, there's no reason that clientless mobs shouldn't count. People can disconnect and be killed while not connected

Reply:
The questionmark is entirely intentional, to check that H exists as well as H's client.